internal/mtail/mtail.go

```diff
@@ -344,7 +345,7 @@ func (m *Server) Run() error {
 			return err
 		}
 	} else {
-		m.store.StartExpiryLoop()
+		m.store.StartExpiryLoop(*m.storeExpireTickInterval)
```

Comment:
This is nil when the test fails, because internal/mtail/basic_tail_integratoin_test.go and others do not initialize the expiration option.  They all could (in internal/mtail/testing.go:MakeTestServer but I think it's better to just handle it, see above.)

Reply:
Thanks I couldn't really figure out where the nill was coming from. I tried to look at the logs or run test locally but didn't have much luck.